admin/howto/manage-users.md

```diff
@@ -50,8 +50,8 @@ specify which users are *authorized* to be on the hub.
 Authorizing regular users
 : Currently, there are only three supported methods for authorizing regular users:
 
-  1. [Manually add users](../howto/manage-users.md) via the admin panel in JupyterHub
-  2. (Google only) Allow all users who are logged in via a particular domain - so
+  1. {ref}`Manually add users<admin/management/admin-panel>` via the admin panel in JupyterHub
```

Comment:
Just a quick Myst tip: you can still use markdown syntax for these, if you use # to reference the target. Eg

```
[Manually add users](#admin/management/admin-panel)
```

Here are some more docs on it 

https://mystmd.org/guide/cross-references#link-references